### PR TITLE
docs: minor grammar & spelling fixes

### PR DESCRIPTION
**Pull request by ckerr**
#### Description of Change

Minor documentation spelling and grammar fixes.

Examples:

 * Typo fix: "Chromiums process model" -> "Chromium's process model"
 * Use 'macOS' instead of 'Mac OS X'
 * Replace mixed use of 'behaviour' and 'behavior' with consistent use of 'behavior' everywhere

CC @electron/wg-ecosystem 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/development/build-instructions-linux.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Follow the guidelines below for building Electron on Linux.
   Doing so permits installing Node on your own home directory as a standard user.
   Or try repositories such as [NodeSource](https://nodesource.com/blog/nodejs-v012-iojs-and-the-nodesource-linux-repositories).
 * [clang](https://clang.llvm.org/get_started.html) 3.4 or later.
-* Development headers of GTK+ and libnotify.
+* Development headers of GTK and libnotify.
```
Comment (malept) [inline docs/development/build-instructions-linux.md]: Nit:

```suggestion
* Development headers of GTK 3 and libnotify.
```

Charles, feel free to disagree as I defer to your judgment 😄

Comment (ckerr) [reply docs/development/build-instructions-linux.md]: My rationale was that GTK 3 came out in 2011, but you're right there are still some tools based on GTK 2 still kicking around out there. :+1:

**Comment by ckerr** [comment]
If they apply cleanly enough for trop to make the change, that's easy enough... I'll add the labels and let's see where it works

File: docs/development/build-instructions-linux.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Follow the guidelines below for building Electron on Linux.
   Doing so permits installing Node on your own home directory as a standard user.
   Or try repositories such as [NodeSource](https://nodesource.com/blog/nodejs-v012-iojs-and-the-nodesource-linux-repositories).
 * [clang](https://clang.llvm.org/get_started.html) 3.4 or later.
-* Development headers of GTK+ and libnotify.
+* Development headers of GTK 3 and libnotify.
```
Comment (zcbenz) [inline docs/development/build-instructions-linux.md]: I think the official name of GTK is `GTK+`?

**Comment by felixrieseberg** [comment]
I like this a lot, consistent style is surprisingly important to me as a reader - and presumably equally noticeable for others. Well done!

File: docs/development/build-instructions-linux.md (hunk 2 above)
Comment (ckerr) [reply docs/development/build-instructions-linux.md]: It was, but the `+` was removed [about a year ago](https://mail.gnome.org/archives/gtk-devel-list/2019-February/msg00000.html)

**Pull request merged**